lib/OrganizationsFieldArray/OrgInfo.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { isEmpty } from 'lodash';
+import { AppIcon } from '@folio/stripes/core';
+import { Field } from 'redux-form';
+import {
+  Button,
+  Col,
+  Card,
+  Layout,
+  Label,
+  Select,
+} from '@folio/stripes/components';
+import { required } from '../validators';
+
+export default class OrgInfo extends React.Component {
+  static propTypes = {
+    index: PropTypes.string,
+    name: PropTypes.string,
+    organization: PropTypes.object,
+    onReplaceField: PropTypes.func,
+    onAddField: PropTypes.func,
+    roles: PropTypes.arrayOf(PropTypes.object),
+  };
+
+  renderOrganizationPicker = (index, name) => {
+    return (
+      <this.props.organizationPickerComponent
+        data-test-org-name
+        onAddOrg={this.onAddOrg}
+        index={index}
+        id={`${name}-nameOrg-${index}`}
+      />
+    );
+  }
+
+  onAddOrg = (organization, index) => {
+    const { name, id } = organization;
+    const org = { name, orgsUuid: id };
+    this.props.onReplaceField(index, { org });
+  }
+
+  unlinkOrg = (index, orgId) => {
+    this.props.onAddField({ id: orgId, _delete: true });
+    this.props.onReplaceField(index, { org: {} });
+  }
+
+  renderOrganizationName = (orgName) => {
+    const { index, name } = this.props;
+    return (
+      <span>
+        <AppIcon
+          app="organizations"
+          appIconKey="organizations"
+          size="small"
+        />
+        {' '}
+        <strong id={`${name}-nameOrg-${index}`}>{`${orgName}`}</strong>
+      </span>
+    );
+  }
+
+  renderOrgInfo = () => {
+    const { index, organization, name } = this.props;
+    const { org } = organization;
+    return (
+      <Card
+        cardStyle="positive"
+        hasMargin
+        headerStart={
+          <strong>Organization</strong>
```

Comment:
i18n. Can you do a pass through this entire PR checking for untranslated strings?